aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -468,7 +468,15 @@ class AztecToolbar : FrameLayout, IAztecToolbar, OnMenuItemClickListener {
         toolbarButtonPlugins.add(buttonPlugin)
 
         val button = findViewById<ToggleButton>(buttonPlugin.action.buttonId)
-        button.setOnClickListener { buttonPlugin.toggle() }
+        val isToolbarAction = ToolbarAction.values().contains(buttonPlugin.action)
```

Comment:
@yuvalgnessin-qz, I noticed Cameron's original comment on this code here was unresolved: https://github.com/wordpress-mobile/AztecEditor-Android/pull/934/files#r571833320

Can you let me know if you addressed the issues raised in that comment somewhere, perhaps I'm missing some context?

Reply:
Yes, this was addressed by ab286ede56b5534212bde3917d112d8c29f97cad

If you comment out `aztec.addPlugin(BackgroundColorButton(visualEditor))` you can see that the BG color button disappears